src/registrar/admin.py

```diff
@@ -137,11 +138,19 @@ class MyUserAdmin(BaseUserAdmin):
         "email",
         "first_name",
         "last_name",
-        "is_staff",
-        "is_superuser",
+        "group",
         "status",
     )
 
+    # Let's define First group
+    # (which should in theory be the ONLY group)
+    def group(self, obj):
+        if obj.groups.filter(name="full_access_group").exists():
+            return "Super User"
```

Comment:
This seems to be inconsistent with how it is mentioned on the rest of the page. "Superuser" instead of "Super User"
![image](https://github.com/cisagov/getgov/assets/141044360/ba6612e9-b911-4eea-a48e-80d462ade510)

If this is a design thing, I'd say that comes first but personally I think Superuser makes more sense in this context for consistency purposes

Reply:
How about 'God mode'?

Reply:
You know, we should remove that superuser filter since it's now obsolete and can be misleading